src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Semantic/Services/DefaultRazorSemanticTokensInfoService.cs

```diff
@@ -24,12 +24,12 @@ namespace Microsoft.AspNetCore.Razor.LanguageServer.Semantic
 {
     internal class DefaultRazorSemanticTokensInfoService : RazorSemanticTokensInfoService
     {
-        // This cache is created to restrict memory growth.
+        // These caches are created to restrict memory growth.
         // We need to keep track of the last couple of requests for use in previousResultId,
-        // but if we let the grow unbounded it could quickly allocate a lot of memory.
-        // Solution: an in-memory cache
-        private readonly MemoryCache<string, (VersionStamp SemanticVersion, IReadOnlyList<int> Data)> _semanticTokensCache =
-            new MemoryCache<string, (VersionStamp SemanticVersion, IReadOnlyList<int> Data)>();
+        // but if we let them grow unbounded it could quickly allocate a lot of memory.
+        // Solution: in-memory caches
+        private readonly MemoryCache<string, (VersionStamp SemanticVersion, IReadOnlyList<int> Data)> _semanticTokensCache = new(); // For RZ docs
```

Comment:
+5 #pranavpoints.

I would suggest spelling out Razor instead of "RZ", I don't think we use that abbreviation anywhere.